samples/extensions/dynamic_state3_multisample_rasterization/dynamic_state3_multisample_rasterization.cpp

```diff
@@ -0,0 +1,876 @@
+/* Copyright (c) 2023, Mobica Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "dynamic_state3_multisample_rasterization.h"
+
+#include "gltf_loader.h"
+#include "scene_graph/components/material.h"
+#include "scene_graph/components/mesh.h"
+#include "scene_graph/components/pbr_material.h"
+#include "scene_graph/components/sub_mesh.h"
+
+DynamicState3MultisampleRasterization::DynamicState3MultisampleRasterization()
+{
+	title = "DynamicState3 Multisample Rasterization";
+
+	set_api_version(VK_API_VERSION_1_1);
+
+	add_instance_extension(VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME);
```

Comment:
You either need `VK_API_VERSION_1_1` or `VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME`. And due to some internally registered extensions that are also promoted to Vulkan 1.1, I would suggest to not set `VK_API_VERSION_1_1`